Assignment-2-User-Management.c

```diff
@@ -0,0 +1,175 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+//defining the struct to store the details of the user
+struct user
+{
+    int id;
+    char name[50];
+    int age;
+};

```

Comment:
for structure, typedef use namingconvention:

Use PascalCase (capitalize the first letter of each word) for struct and typedef names.
Use descriptive names that reflect the data structure's purpose.

typedef struct Student {
int id;
char name[50];
float grade;
} Student;

Use PascalCase (capitalize the first letter of each word) for struct and typedef names.
Use descriptive names that reflect the data structure's purpose.

ex:
typedef struct Student {
int id;
char name[50];
float grade;
} Student;

Student student_data;